node_modules/oae-preview-processor/lib/api.js

```diff
@@ -507,11 +507,13 @@ var _generatePreviews = function(ctx, callback) {
         if (err) {
             return callback(err);
         } else if (processor) {
-            processor.generatePreviews(ctx, ctx.content, function(err, ignored) {
+            processor.generatePreviews(ctx, ctx.content, function(err, ignored, pending) {
                 if (err) {
                     ctx.setStatus('error', callback);
                 } else if (ignored) {
                     ctx.setStatus('ignored', callback);
+                } else if (pending) {
+                    ctx.setStatus('pending', callback);
                 } else {
```

Comment:
Can you extract these `'error'`, `'ignored'` and `'pending'` into an enumeration-like thing in a `constants.js` file? That way usages can be better tracked for some analysis when adding new ones like this.